src/main/java/reposense/parser/CsvParser.java

```diff
@@ -58,7 +61,14 @@ public CsvParser(Path csvFilePath) throws IOException {
             String line;
 
             for (int lineNumber = 2; (line = br.readLine()) != null; lineNumber++) {
-                String[] elements = line.split(ELEMENT_SEPARATOR);
+                String[] elements = line.split(ELEMENT_SEPARATOR_REGEX);
+                for (int colNum = 0; colNum < elements.length; colNum++) {
+                    Matcher matcher = DOUBLEQUOTE_DETECTOR_REGEX.matcher(elements[colNum]);
```

Comment:
Can explain what is `DOUBLEQUOTE_DETECTOR_REGEX` matching?

Reply:
Matching fields that contain either double quotes or commas. Fields containing `"` or `,` will end up having double-quotes added to the beginning and the end of field.

So  `Ben " S`  --> `"Ben "" S"` and also `Ben , S` --> `"Ben , S"`

Reply:
Then do you really need regex for this?

i.e. Can directly use `StringsUtil#removeQuote`, if required test the output.

Reply:
Nope no need for regex. Thought too much about filtering off those characters.